### PR TITLE
create book model, service, repository ,changed pom.xml 

### PR DESCRIPTION
**Pull request by JeniaSan**
and application.properties

### CODE REVIEW

**Review by ConfusedSatlan — APPROVED** [review]
good

File: src/main/java/com/polishuchenko/bookstore/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    @Column(nullable = false)
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
+
```
Comment (nacenik) [inline src/main/java/com/polishuchenko/bookstore/model/Book.java]:
```suggestion
```

File: src/main/java/com/polishuchenko/bookstore/repository/impl/BookRepositoryImpl.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class BookRepositoryImpl implements BookRepository {
+    @Autowired
+    private SessionFactory sessionFactory;
```
Comment (nacenik) [inline src/main/java/com/polishuchenko/bookstore/repository/impl/BookRepositoryImpl.java]: Better make fields `final`
fix it everywhere

File: src/main/java/com/polishuchenko/bookstore/repository/impl/BookRepositoryImpl.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            }
+        }
+    }
+
+    @Override
+    public List<Book> findAll() {
+        try (Session session = sessionFactory.openSession()) {
+            return session.createQuery("FROM Book ", Book.class).getResultList();
+        } catch (Exception e) {
+            throw new RuntimeException("Can't get all books");
```
Comment (nacenik) [inline src/main/java/com/polishuchenko/bookstore/repository/impl/BookRepositoryImpl.java]: Throwing a general RuntimeException is not best practice.

**Review by nacenik — CHANGES_REQUESTED** [review]
Good solution, let's improve it :)

**Review by nacenik — APPROVED** [review]
LGTM! 👍

**Pull request merged**